python/_nimblephysics/simulation/World.cpp

```diff
@@ -296,12 +296,20 @@ void World(py::module& m)
           "runConstraintEngine",
```

Comment:
What's the difference between `runConstraintEngine()` and `runLcpConstraintEngine()`? It's not immediately obvious from reading it, so maybe we should make the names a bit more explicit?